src/service/core-services.js

```diff
@@ -125,7 +129,15 @@ function installAmpdocServicesInternal(ampdoc, isEmbedded) {
   isEmbedded
     ? adoptServiceForEmbedDoc(ampdoc, 'history')
     : installHistoryServiceForDoc(ampdoc);
-  const fieResourcesOn = isFieResourcesOn(ampdoc.win);
+
+  // fie-resources experiment.
+  if (ampdoc.isSingleDoc()) {
+    divertFieResources(ampdoc.win);
+  }
+  const fieResourcesOn =
+    ampdoc.getParent() &&
```

Comment:
is it possible for this function to be called for the embed before the parent?

Reply:
by this function I mean `installAmpdocServicesInternal`

Reply:
No. Parent is always initialized first.